ewatercycle/models/wflow.py

```diff
@@ -15,7 +15,7 @@
 from ewatercycle.models.abstract import AbstractModel
 from ewatercycle.parameter_sets import ParameterSet
 from ewatercycle.parametersetdb.config import CaseConfigParser
-from ewatercycle.util import get_time
+from ewatercycle.util import get_time, to_absolute_path
 
 
 class Wflow(AbstractModel[WflowForcing]):
```

Comment:
For line 59 spawned of https://github.com/eWaterCycle/ewatercycle/issues/144

Reply:
It will be fixed in another PR.